static/static.go

```diff
@@ -0,0 +1,195 @@
+//go:generate env GO111MODULE=on go run github.com/kevinburke/go-bindata/go-bindata -o static_gen.go -ignore 'map|go' -tags assets -pkg static data/...
```

Comment:
For a follow-up, I wonder if this module could help us drop `go-bindata` entirely: https://github.com/vearutop/statigz

Reply:
Oh cool, that does look nice, especially since it claims to also set etags. I added a note to the `go:embed` issue for future reference: https://github.com/influxdata/influxdb/issues/21652